### PR TITLE
Telemetry implemented for Certificate download, profile dropdown menu, and fix get-started

### PR DESCRIPTION
**Pull request by garima468**
Telemetry implemented for Certificate download, profile dropdown menu, and fix get-started

### CODE REVIEW

File: library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html [hunk 1]
```diff
@@ -84,14 +84,14 @@ <h2 class="mat-title margin-bottom-xs" i18n>Pinned</h2>
       <!-- <mat-icon>settings</mat-icon> -->
       <span i18n>'Get Started' tour</span>
     </a>
-    <a mat-menu-item role="link" (click)="redirectToMyLearning()" class="app-features flex items-center">
+    <a mat-menu-item role="link" (click)="redirectToMyLearning();" class="app-features flex items-center">
```
Comment (venkykandagaddala) [inline library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html]: @garima468 Please remove the semicolon else build will fail

File: library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html [hunk 2]
```diff
@@ -207,7 +207,7 @@ <h4 class="mat-h4 margin-bottom-remove nav_sub_heading">Coming soon</h4>
 
 <ng-template #userIndividualNew let-hubs="hubs">
   <ng-container *ngIf="hubs.hubname == 'Discuss' && !hubs.active">
-    <a [id]="hubs.hubname" class="mobile flex flex-wrap mb-4" (click)="toggleVisibility(); navigate()" role="menuitem" i18n-aria-label>
+    <a [id]="hubs.hubname" class="mobile flex flex-wrap mb-4" (click)="toggleVisibility(); navigate(); trackTelemetry(hubs.hubname)" role="menuitem" i18n-aria-label>
```
Comment (venkykandagaddala) [inline library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html:210]: Remove the semicolon else the build will fail.

File: library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html [hunk 3]
```diff
@@ -223,7 +223,7 @@ <h4 class="mat-h4 margin-bottom-remove nav_sub_heading">Coming soon</h4>
   </ng-container>
 
   <ng-container *ngIf="hubs.hubname != 'Discuss' && !hubs.active">
-    <a class="flex flex-wrap mb-4 mobile" [id]="hubs.hubname" (click)="toggleVisibility();" [routerLink]="(hubs.path)" role="menuitem" i18n-aria-label>
+    <a class="flex flex-wrap mb-4 mobile" [id]="hubs.hubname" (click)="toggleVisibility(); trackTelemetry(hubs.hubname)" [routerLink]="(hubs.path)" role="menuitem" i18n-aria-label>
```
Comment (venkykandagaddala) [inline library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html:226]: Remove the semicolon else the build will fail.

File: project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html [hunk 4]
```diff
@@ -357,7 +357,7 @@ <h3 class="margin-remove-bottom pt-4 info-msg" i18n *ngIf="content?.status === '
             <span class="certificate-text block">If the issue persists, please mail us at <a>mission[dot]karmayogi[at]gov[dot]in</a></span>
           </ng-container>
           <ng-container *ngIf="certData">
-            <a class="certificate_box" (click)="openCertificateDialog()">
+            <a class="certificate_box" (click)="openCertificateDialog(); raiseCertIntreactTelemetry()">
```
Comment (venkykandagaddala) [inline project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html:360]: Remove the semicolon else the build will fail.

File: library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html (hunk 1 above)
Comment (garima468) [reply library/ws-widget/collection/src/lib/btn-profile/btn-profile.component.html]: removed

File: library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html (hunk 2 above)
Comment (garima468) [reply library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html:210]: I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.

File: library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html (hunk 3 above)
Comment (garima468) [reply library/ws-widget/collection/src/lib/card-hubs-list/card-hubs-list.component.html:226]: I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.

File: project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html (hunk 4 above)
Comment (garima468) [reply project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html:360]: I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.

**Review by venkykandagaddala — APPROVED** [review]

**Pull request merged**